citizennet/purescript-ocelot
# Move build to `make`

## What does this pull request do?

We want to decrease the amount of friction required for working on this library. Having to remember a variety of commands necessary to get started and to build the project is one source of friction we can address. 

## How should this be manually tested?

If you don't have a local deployment of Ocelot, I say don't worry about it, but if you'd like:
- Run `make clean`.
- [x] Verify you get a blank screen when you load your local deployment of Ocelot.
- Run `make`.
- [x] Verify the Ocelot UI Guide loads as expected.

## Review assignment

There's an ongoing [Slack thread](https://citizennet.slack.com/archives/C3Q0ATZST/p1616465505002100) concerning how best to assign reviews for this and other open source PS repos. Since that conversation is not yet resolved, I'm just going to assign this to someone from each team that uses Ocelot. Anyone should feel comfortable removing themselves or re-assigning the PR if for whatever reason they don't wish to review it.
